sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/TungstenAggregate.scala

```diff
@@ -69,6 +72,9 @@ case class TungstenAggregate(
   protected override def doExecute(): RDD[InternalRow] = attachTree(this, "execute") {
     val numInputRows = longMetric("numInputRows")
     val numOutputRows = longMetric("numOutputRows")
+    val totalPeakMemory = longMetric("totalPeakMemory")
```

Comment:
Also it's weird to say "total" in some places but not in others. For instance, records in and out are also totals, but it doesn't say "total" in those.